resource-watch/resource-watch
# Update all routes (header, footer, dashboard gallery) to dashboard dynamic routes (from API). Removed topics routes.

## Overview
This PR is the second part to move topics to dashboards:
1) This PR updates the routes for dashboards gallery. Topics routes removed from dashboards page and menus.
[Pivotal task](https://www.pivotaltracker.com/n/projects/1374154/stories/170311468)
2)  This PR updates the links in header and footer. Topic links were changed to feature dashboards links.
[Pivotal task](https://www.pivotaltracker.com/n/projects/1374154/stories/170311607)
3) This PR updates API requests for featured dashboard and dashboard gallery. Added is-featured and is-highlighted filter options to requests. (updated requests and redux state).
[Pivotal task](https://www.pivotaltracker.com/n/projects/1374154/stories/170311599)
[Pivotal task](https://www.pivotaltracker.com/n/projects/1374154/stories/169732990)
[Pivotal task](https://www.pivotaltracker.com/n/projects/1374154/stories/170397448)
4) This PR updates visual part of dashboard gallery.
[Pivotal task](https://www.pivotaltracker.com/n/projects/1374154/stories/169732990)
5) This PR removes and renames old topics files.
[Pivotal task](https://www.pivotaltracker.com/n/projects/1374154/stories/170311546)
## Testing instructions
Use staging API
1) Footer and Header links:
 - Go to header menu, hover dashboards, click on dashboard.
 - Go to footer menu click on dashboard.
You should go to /dashboards/:slug page
2) Gallery links:
 - Go to /dashboads page. Click on gallery item. You should go to /dashboards/:slug page
3) Featured links
 - Go to /dashboads page. Click on item in featured dashboads list. You should go to /dashboards/:slug page
The featured option has already added to a request (for links and featured dashboards). But is not deployed on staging on BE side. We shoul wait utill this option will be activated.
4) Is-highlighted option. This option was added on staging by previous PR. And added to admin dashboard (new/edit) pages too. So, to add items to dashboard gallery go to /admin/dashboards select or create dashboard and set `is-highlighted` and `published` flags
---

## Checklist before submitting
